css/styles.css

```diff
@@ -0,0 +1,48 @@
+/* possible color theme
+ * #4169E1, #C0C0C0, #272440, #2F2C40, #0D0D0D */
+p {
+  font-family: 'Syncopate', sans-serif;
+  padding: 5px;
+  margin: 10px;
+}
+body {
+  background-color: #C0C0C0;
+}
+h1 {
+  font-family: 'Syncopate', sans-serif;
+  color: #4169E1;
+  padding: 5px;
```

Comment:
I see some consistent attribs, like padding and width. Could you abstract those into one class, and apply that class to your HTML to simplify the CSS?